api/proto/disperser/v2/disperser_v2.proto

```diff
@@ -73,10 +73,15 @@ message BlobStatusRequest {
 message BlobStatusReply {
   // The status of the blob.
   BlobStatus status = 1;
-  // The signed batch. Unset if the status is not CERTIFIED.
+  // The signed batch. Only set if the blob status is GATHERING_SIGNATURES or COMPLETE.
+  // signed_batch and blob_inclusion_info are only set if the blob status is GATHERING_SIGNATURES or COMPLETE.
+  // When blob is in GATHERING_SIGNATURES status, the attestation object in signed_batch contains attestation information
+  // at the point in time. As it gathers more signatures, attestation object will be updated according to the latest attestation status.
+  // The client can use this intermediate attestation to verify a blob if it has gather enough signatures. 
+  // Otherwise, it should should poll the GetBlobStatus API until the desired level of attestation has been gathered or status is COMPLETE.
```

Comment:
Prob want to add that if required quorum thresholds have not been met, a new dispersal will need to be sent.